### ISSUE TITLE
Ajax Control Toolkit 16.1 Improper display TabContainer.

### ISSUE THREAD

**Issue by LorDLeLiK** [DevExpress/AjaxControlToolkit#48]
Hi,
When you create a site to .Net FrameWork 4.0 (VS 2015) at local testing site (on IIS-express) control TabContainer displayed normally. But after loading the created site on the server (on IIS 7), there is a distortion of the appearance of the control, control functionality (switching tabs) works fine. IE 11 browser.
PS I apologize for my English.
![image distortion](https://cloud.githubusercontent.com/assets/18218909/14216776/42611a78-f853-11e5-9cca-a121455bdb02.png)

**Comment by MikhailTymchukDX** [comment]
CSS style for Tabs were changed in v16.1, so you need to apply the workaround described here: https://github.com/DevExpress/AjaxControlToolkit/issues/5#issuecomment-196755356 and https://github.com/DevExpress/AjaxControlToolkit/issues/5#issuecomment-197203997

**Comment by qunshen** [comment]
I experiencing the same problem!

**Comment by MikhailTymchukDX** [comment]
@qunshen Did you try to apply the described workaround?

**Comment by winshell** [comment]
Hi, 

same problem here. Tried the workaround, tried the sample in that post. Did not work.

Scenario1:
Local Machine: Ajax 16.1.0, your sample, Win7 SP1, VS2015 SP1, IIS 10.0 Express --> works
Server: Ajax 16.1.0, your sample, Server Win 2012 R2, IIS 8.5 --> does not work

Scenario2:
Local Machine: Ajax 15.1.4, your sample, Win7 SP1, VS2015 SP1, IIS 10.0 Express --> works
Server: Ajax 15.1.4, your sample, Server Win 2012 R2, IIS 8.5 --> works

Could you have another look at this topic. Thanks.

**Comment by MikhailTymchukDX** [comment]
@winshell My sample runs fine on Windows Server 2012 R2:
![image](https://cloud.githubusercontent.com/assets/17080984/14850884/c01a2e6a-0c84-11e6-8f2f-87349cd6aa2f.png)

Would you please provide a sample screenshot and [export IIS settings](http://www.sherweb.com/blog/using-iis-shared-configuration-to-migrate-your-sites-to-another-server/) to apply it on my server?

**Comment by winshell** [comment]
![untitled](https://cloud.githubusercontent.com/assets/18679930/14889276/79232e5a-0d5e-11e6-899b-99f1ba6d55cc.png)
[TestApp.zip](https://github.com/DevExpress/AjaxControlToolkit/files/240722/TestApp.zip)

Please find attached the sample that did not work.

I need to correct my statement on Scenario1 from the previous post: Scenario1 does not work on local machine as well as not on the Server.

Unclear to me, if this is a CSS config problem or an AJAX problem. The sample is working with 15.1.4

**Comment by MikhailTymchukDX** [comment]
@winshell There are two rules in the CSS declaration, but you changed only the first one.
Replace `tabs` with `MyTabStyle` to fix this:
```css
.MyTabStyle .ajax__tab_header:before, .MyTabStyle .ajax__tab_header:after {
    content: "";
    display: table;
}
```

**Comment by winshell** [comment]
Sorry for confusion. Sometimes thoughtful reading of information helps....Shame on my head!

My problem is solved. Works like a charm with 16.1.0.0

**Comment by winshell** [comment]
A similar problem/question.

One of my user still had the body on the right side as shown in my sample image. First, we did not understand. After a while we figured out that he added our intranet website to the "Compatibility View Settings" list. We are using IE 11.0.9600. When we removed the intranet website, it was working again. He said, he needs the "Compatibility View" for some reason.

Any idea how to fix that my attached sample also works with IE 11.0.9600 in "Compatibility View" ?

Thanks.
